ifupdown2/addons/bond.py

```diff
@@ -554,6 +576,58 @@ def get_ifla_bond_attr_from_user_config(self, ifaceobj, link_exists):
         self._check_bond_mode_user_config(ifname, link_exists, ifla_info_data)
         return ifla_info_data
 
+    def check_miimon_arp(self, link_exists, ifaceobj, ifla_info_data):
+        """
+            Check bond checks either miimon or arp ip check
+        """
+        # miimon
+        ifla_bond_miimon = ifla_info_data.get(Link.IFLA_BOND_MIIMON)
+        if link_exists and ifla_bond_miimon is None:
+            ifla_bond_miimon = self.cache.get_link_info_data_attribute(ifaceobj.name, Link.IFLA_BOND_MIIMON)
+
+        if ifla_bond_miimon:
+            ifla_bond_miimon = str(ifla_bond_miimon)
```

Comment:
I do it because the config attribute is string 'balance-rr', 'balance-xor' and the netlink attribute is int 0, 1, 2...
In order to cover both cases I converted the netlink attribute to string so I can do:

if ifla_bond_mode not in ['0', '2', 'balance-rr', 'balance-xor']

Do you want me to split the cases?